### ISSUE TITLE
fileFormatVersion and vendorName tags not being captured from XML

### ISSUE THREAD

**Issue by jsanz01** [bodastage/boda-measdatacollectionparser#8]
Hi,
I have started to use measdatacollectionparser and I have noticed that the resulting CSV file is not including fileFormatVersion and vendorName tags (those columns are empty). After taking a look to the source code and comparing with the 3GPP XML Specification, in my opinion those tags are located at fileHeader area, however the code is trying to find them at fileSender area and is not being success and it is the reason.

localDN attribute is just the opposite (it belongs to fileSender), but because fileSender is a subset of fileHeader, it still works.

Best regards, Jose

```
if (tag == "fileHeader") {
            for (m <- attrs) {
              if (m.key == "localDn") fileSenderLocalDn = m.value.toString()
              if (m.key == "elementType") elementType = m.value.toString()
              if (m.key == "dnPrefix") fileHeaderDnPrefix = m.value.toString()
            }
          }

          if (tag == "fileSender") {
            for (m <- attrs) {
              if (m.key == "fileFormatVersion") fileFormatVersion = m.value.toString()
              if (m.key == "vendorName") vendorName = m.value.toString()
            }
          }
```

**Comment by erssebaggala** [comment]
@jsanz01, I have checked and you are very correct.  Let me correct that.

**Comment by erssebaggala** [comment]
Resolved by PR #8
